pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -26,104 +25,72 @@ import MultiSelect from "../common/multiSelect";
 import ElementCreationNew from "../common/elementCreationNew";
 
 interface HandlerFormProps {
-  id: string;
+  handlerId: string;
```

Comment:
You've changed this prop key, but you have not changed the key when passing it to this component.

Meaning within `[handlerId].tsx`: 

Current: `<HandlerForm id={handlerId} endpointId={endpointId} />`

Target: `<HandlerForm {... {handlerId, endpointId}} />`